modules/ROOT/pages/security/configuration.adoc

```diff
@@ -94,7 +165,9 @@ If the previous `roles` claim is not located at the JWT payload root, but instea
 }
 ----
 
-This needs to be configured using the `@jwtClaim` directive:
+// ^ why is this a nested location? can we show the nesting?
```

Comment:
^

Reply:
nested in this case refers to a nested object in the JSON structure.

In this case `myApplication` is at the top-level, while `roles` is _nested_ under `myApplication`

```json
    "myApplication": {
        "roles": ["user", "admin"]
    }
```